### PR TITLE
Frontend Modularization: Dashboards

### PR DESCRIPTION
**Pull request by maryamkhidir**
### Feature or Bugfix

- Refactoring

### Detail
- Modularized Dashboards folder into views, components and services subfolders
- Refactored relative imports used in the module to use absolute paths
- Removed unused scripts: `services/graphql/Dashboard/createDashboard.js` and `services/graphql/Dashboard/shareDashboard.js`
- Updated routes to reflect new file paths

### Relates
- https://github.com/awslabs/aws-dataall/issues/398

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/src/modules/Shared/Feed/FeedCommentAdd.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import SendIcon from '@mui/icons-material/Send';
```
Comment (nikpodsh) [inline frontend/src/modules/Shared/Feed/FeedCommentAdd.js:1]: The PR should be rebased on top of the fresh main. `modularization/frontend` has `FeedCommentAdd.js` under different directory

Comment (maryamkhidir) [reply frontend/src/modules/Shared/Feed/FeedCommentAdd.js]: PR updated with https://github.com/awslabs/aws-dataall/pull/600/commits/38c2e340ee3ac577c5214267aaabfc7a3f302f00

File: frontend/src/routes.js [hunk 2]
```diff
@@ -101,19 +101,19 @@ const MLStudioCreateForm = Loadable(
 );
 
 const DashboardList = Loadable(
-  lazy(() => import('./modules/Misc/views/Dashboards/DashboardList'))
+  lazy(() => import('./modules/Dashboards/views/DashboardList'))
```
Comment (nikpodsh) [inline frontend/src/routes.js:104]: We should add `config.modules.dashboards.active` here and in `DefaultSidebar.js`
You can use `config.modules.notebooks.active` as reference

Comment (maryamkhidir) [reply frontend/src/routes.js:104]: Updated with [f84d758](https://github.com/awslabs/aws-dataall/pull/600/commits/f84d758a1514407246bb08898630ebbd07843928)

**Pull request merged**